jobs/JGFS_ATMOS_GEMPAK

```diff
@@ -31,78 +31,85 @@ export DBN_ALERT_TYPE=${DBN_ALERT_TYPE:-GFS_GEMPAK}
 # Specify NET and RUN Name and model
 ####################################
 export model=${model:-gfs}
-export COMPONENT="atmos"
 
 ##############################################
 # Define COM directories
 ##############################################
-export COMIN=${COMIN:-$(compath.py ${envir}/${NET}/${gfs_ver})/${RUN}.${PDY}/${cyc}/${COMPONENT}}
-export COMOUT=${COMOUT:-$(compath.py -o ${NET}/${gfs_ver}/${RUN}.${PDY})/${cyc}/${COMPONENT}/gempak}
-
 export SENDDBN=${SENDDBN:-NO}
 export DBNROOT=${DBNROOT:-${UTILROOT}/fakedbn}
 
-if [ ${SENDCOM} = YES ] ; then
-  mkdir -m 775 -p ${COMOUT}
+for res in 0p25 0p50 1p00; do
+  RES=${res} YMD=${PDY} HH=${cyc} generate_com -rx "COM_ATMOS_GRIB_${res}:COM_ATMOS_GRIB_TMPL"
+done
+
+YMD=${PDY} HH=${cyc} generate_com -rx COM_ATMOS_GEMPAK
+
+if [[ ${SENDCOM} = YES && ! -d "${COM_ATMOS_GEMPAK}" ]] ; then
+  mkdir -m 775 -p "${COM_ATMOS_GEMPAK}"
 fi
 
 
 # TODO: These actions belong in an ex-script not a j-job
-if [ -f poescript ]; then
+if [[ -f poescript ]]; then
    rm -f poescript
 fi
 
+ocean_domain_max=180
+if (( ocean_domain_max > FHMAX_GFS )); then
+  ocean_domain_max=${FHMAX_GFS}
+fi
+
 #################################################################
 # Execute the script for the 384 hour 1 degree grib
 ##################################################################
-echo "time ${SRCgfs}/exgfs_atmos_nawips.sh gfs 384 GFS_GEMPAK &> ${DATA}/gfs_1p0.$$.1 " >>poescript
-echo "time ${SRCgfs}/exgfs_atmos_nawips.sh gfs 384 GFS_GEMPAK &> ${DATA}/gfs_1p0.$$.2 " >>poescript
-echo "time ${SRCgfs}/exgfs_atmos_nawips.sh gfs 384 GFS_GEMPAK &> ${DATA}/gfs_1p0.$$.3 " >>poescript
-echo "time ${SRCgfs}/exgfs_atmos_nawips.sh gfs 384 GFS_GEMPAK &> ${DATA}/gfs_1p0.$$.4 " >>poescript
-echo "time ${SRCgfs}/exgfs_atmos_nawips.sh gfs 384 GFS_GEMPAK &> ${DATA}/gfs_1p0.$$.5 " >>poescript
-echo "time ${SRCgfs}/exgfs_atmos_nawips.sh gfs 384 GFS_GEMPAK &> ${DATA}/gfs_1p0.$$.6 " >>poescript
+echo "time ${SRCgfs}/exgfs_atmos_nawips.sh gfs ${FHMAX_GFS} GFS_GEMPAK &> ${DATA}/gfs_1p0.$$.1 " >> poescript
```

Comment:
Could these be looped through by resolution?

Reply:
Beyond the scope of this PR. The gempak/awips scripts in general could do with a lot of rewriting.